dynamodb/src/test/java/docs/javadsl/RetryTest.java

```diff
@@ -50,15 +47,10 @@ public void setup() throws Exception {
             // #awsRetryConfiguration
             .overrideConfiguration(
                 ClientOverrideConfiguration.builder()
-                    .retryPolicy(
-                        // This example shows the AWS SDK 2 `RetryPolicy.defaultRetryPolicy()`
-                        // See
-                        // https://sdk.amazonaws.com/java/api/latest/software/amazon/awssdk/core/retry/RetryPolicy.html
-                        RetryPolicy.builder()
-                            .backoffStrategy(BackoffStrategy.defaultStrategy())
-                            .throttlingBackoffStrategy(BackoffStrategy.defaultThrottlingStrategy())
-                            .numRetries(SdkDefaultRetrySetting.defaultMaxAttempts())
-                            .retryCondition(RetryCondition.defaultRetryCondition())
+                    .retryStrategy(
+                        // See https://sdk.amazonaws.com/java/api/latest/software/amazon/awssdk/retries/api/RetryStrategy.html
+                        DefaultRetryStrategy.legacyStrategyBuilder()
+                            .treatAsThrottling(e -> true)
```

Comment:
Better with a logging too？

Reply:
I merged this because of the broken build.
I don't understand what the logging requirement is. This is test code used in the docs. What sort of logging would you like to see?